reason-react-native/src/apis/AsyncStorage.md

```diff
@@ -8,47 +8,39 @@ wip: true
 type error;
 
 [@bs.scope "AsyncStorage"] [@bs.module "react-native"]
-external getItem: string => Js.Promise.t(Js.Null.t(string)) = "getItem";
-let getItem = key =>
-  getItem(key) |> Js.Promise.then_(PromiseUtils.nullableToOption);
+external getItem: string => Js.Promise.t(Js.Null.t(string)) = "";
 
 [@bs.scope "AsyncStorage"] [@bs.module "react-native"]
-external setItem: (string, string) => Js.Promise.t(unit) = "setItem";
+external setItem: (string, string) => Js.Promise.t(unit) = "";
 
 [@bs.scope "AsyncStorage"] [@bs.module "react-native"]
-external removeItem: string => Js.Promise.t(unit) = "removeItem";
+external removeItem: string => Js.Promise.t(unit) = "";
 
 [@bs.scope "AsyncStorage"] [@bs.module "react-native"]
-external mergeItem: (string, string) => Js.Promise.t(unit) = "mergeItem";
+external mergeItem: (string, string) => Js.Promise.t(unit) = "";
 
 [@bs.scope "AsyncStorage"] [@bs.module "react-native"]
-external clear: unit => Js.Promise.t(unit) = "clear";
+external clear: unit => Js.Promise.t(unit) = "";
 
 [@bs.scope "AsyncStorage"] [@bs.module "react-native"]
-external getAllKeys: unit => Js.Promise.t(Js.Null.t(array(string))) =
-  "getAllKeys";
-let getAllKeys = () =>
-  getAllKeys() |> Js.Promise.then_(PromiseUtils.nullableToOption);
+external getAllKeys: unit => Js.Promise.t(Js.Null.t(array(string))) = "";
 
 [@bs.scope "AsyncStorage"] [@bs.module "react-native"]
 external flushGetRequests: unit => unit = "";
 
 [@bs.scope "AsyncStorage"] [@bs.module "react-native"]
 external multiGet:
   array(string) => Js.Promise.t(Js.Null.t(array(array(string)))) =
-  "multiGet";
-let multiGet = keys =>
-  multiGet(keys) |> Js.Promise.then_(PromiseUtils.nullableToOption);
+  "";
```

Comment:
Ah, I wasn't aware we were still using the PromiseUtils module. Could you remove it, too?

Reply:
`PromiseUtils` is long gone, but this doc seems to have become `async`

Reply:
Ah, ok! 😉